src/Authorization/Models/EventLog/AuthorizationEvent.cs

```diff
@@ -16,7 +16,7 @@ public class AuthorizationEvent
         /// <summary>
         /// Date, time of the authorization event. Set by producer of logevents
         /// </summary>
-        public DateTime Created { get; set; }
+        public DateTimeOffset Created { get; set; }
```

Comment:
This should probably be `DateTimeOffset?`. Otherwise you have no good way of detecting if it wasn't set.